creation/web_base/singularity_lib.sh

```diff
@@ -1593,7 +1621,7 @@ singularity_locate_bin() {
             test_out=$(singularity_test_bin "${s_step},${s_location}/$test_out" "$s_image") &&
                 HAS_SINGULARITY=True
             bread_crumbs+="${test_out##*@}"
-        elif [[ ! ",PATH,CONDOR,OSG," = *",$s_location,"* ]]; then  # Don't print error for keywords
+        elif [[ ! ",PATH,CONDOR,OSG,," = *",$1,"* ]]; then  # Don't print error for keywords or empty value
```

Comment:
Is the extra comma at the end intentional in `",PATH,CONDOR,OSG,,` ?

Reply:
yes, for empty value